ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,72 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
+- fire_alarm_status
+- fire_fault_status
+- makeup_water_high_tank_percentage_status
```

Comment:
if its a percentage it needs to be a `sensor`

Reply:
It is a digital signal, I will remove the "percentage" . It will become "makeup_water_high_tank_status"